projects/js-packages/ai-client/src/components/ai-image/components/carrousel.scss

```diff
@@ -36,7 +36,7 @@ $scale-factor: 0.55;
 
 		&-message {
 			font-size: 10px;
-			color: red;
+			color: #ff2c2c;
```

Comment:
Why `#ff2c2c` rather than `#f00` as shown at https://drafts.csswg.org/css-color-4/#valdef-color-red?

Same in a few other places in here.

Reply:
Probably because I was looking at a bad list. 🤔 I think this is what I looked at:
https://www.figma.com/colors/red/

I'll fix these on Monday.